src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,23 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private ReaderOurCsv reader = new ReaderOurCsv();
+    private MakerStatistic makerStatistic = new MakerStatistic();
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        try (BufferedWriter bufferedWriter =
+                     new BufferedWriter(new FileWriter(new File(toFileName), true))) {
+            for (String record : makerStatistic
+                    .makeStatistic(reader.getArrayRecords(fromFileName))) {
+                bufferedWriter.write(record);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write info to file", e);
+        }
     }
```

Comment:
It would be cool to create a separate class that will write data to a file, just as you did with ReaderOurCsv